src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -608,14 +609,28 @@ public void processRedirectUrl(final WebView view, String url){
         }
         
         public boolean processInvalidUrl(final WebView view, String url) {
+        	final String methodName = ".processInvalidUrl";
```

Comment:
nitpick: can you change the . to :? The whole code I think it's using :, we should make it consistent. If we want to change it later, then it should be across file changes.